src/pages/Lobby/components/CourtsSection/types.ts

```diff
@@ -0,0 +1,7 @@
+import { Player } from '@shared/types';
+
+type Court = Player[];
+
+export interface CourtsSectionProps {
+  courts: Court[];
```

Comment:
Do you think this type makes sense? I tried to define something that would work for one or more courts

Reply:
I agree with this modeling! Congrats